docs/visualization/plotting-overview.ipynb

```diff
@@ -202,28 +246,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d.coords['xlabels'] = sc.Variable(dims=['x'], values=np.arange(M) + 15.)\n",
-    "sc.plot(d['data_2d'], labels={'x': 'xlabels'})"
+    "da.transpose().plot()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<div class=\"alert alert-info\">\n",
-    "\n",
-    "*Note*\n",
-    "\n",
-    "The `labels` dict uses *dimension labels* as keys, i.e., the `'x'` above refers to the `x` dimension and *not* the x-axis of the matplotlib figure.\n",
-    "\n",
-    "</div>"
+    "When the Matplotlib \"widget\" backend is activate it is also possible to transpose the axes via a button in the toolbar."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This also works for attributes:"
+    "## Logarithmic scale\n",
+    "\n",
+    "1-D data can be plotted on a logarithmic scale on one or both axes.\n",
```

Comment:
This reads as if it's only possible on 1d plots, but you can also do it with 2d plots. Should we also have a 2d example?